### DISCUSSION TITLE
2.3.2 burn down

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#11234] [General]
@mattpap What if anything is remaining before we can cut 2.3.2? I would like to finish the `OpenURL` if the test issues can be sorted, is there anything blocking on your end?

**Comment by mattpap** [comment]
I don't have anything to add to this release right now. Layout regression fixes are work in progress (including the new #11231).

**Reply by bryevdv** [reply]
OK I will shoot for wednesday (tomorrow if I can)
